src/pages/sources.vue

```diff
@@ -162,8 +162,12 @@ const SourcePage = {
     },
   },
   methods: {
+    /**
+     * @param {number} imageCount
+     */
     getProviderImageCount(imageCount) {
-      return imageCount.toLocaleString(this.$i18n.locale)
+      // Always use EN, most sites with RTL language with numbers continue to use Western Arabic Numerals whereas `toLocaleString` will use Eastern Arabic Numerals for Arabic and Hebrew by default
+      return imageCount.toLocaleString('en')
```

Comment:
I wonder if we should check for Arabic specifically, because `toLocaleString(en)` uses a comma as a thousands separator, and some European languages use comma as decimal separator, and a period or a space for thousands separator:
```
const num = 25000
num.toLocaleString('en')
"25,000"
num.toLocaleString('de')
"25.000"
num.toLocaleString('ru')
"25 000"
```

Reply:
^+1

Reply:
Oh yes definitely, good call. Hebrew was also using Eastern Arabic Numerals, but I solved this problem previously for Gutenberg so I will find that PR and apply a similar logic here. I can't believe I forgot about that!

Reply:
Nice! Really cool that you have prior experience fixing that issue.